scripts/prepare_test_env.sh

```diff
@@ -30,6 +30,14 @@ export REQUESTS_CA_BUNDLE=`pwd`/ca.pem
 export MANAGEMENT_CA_CERT_PATH=`pwd`/ca.pem
 export CURL_CA_BUNDLE=`pwd`/ca.pem
 
+
+if [ "$MGT_API_AUTHORIZATION" == "true" ]; then
+  USER_PASSWD=`kubectl get secret imm-openldap-customldif -o yaml|grep immconfig|awk '{ print $2 }'|base64 --decode|grep userpassword|awk '{ print $2 }'`
+  USER_NAME=user@example.com
+  export SINGLE_TENANT_USER=$USER_NAME:$USER_PASSWD
```

Comment:
Perhaps this name could indicate what it consists of. USER_CREDENTIALS or something like that.